### ISSUE TITLE
Is the code consistent with the description in the paper?

### ISSUE THREAD

**Issue by AlphaPlusTT** [Kevinz-code/CSRA#5]
According to formula 5 and formula 6 in the paper, the class-specific residual attention (CSRA) feature _f_ should be sent to
the classifier to obtain the final logits, but in your code, you use the _f_ as the final logits, what's the difference?

**Comment by Kevinz-code** [comment]
Hi,  @AlphaPlusTT 

Actually there are no difference and the code is consistent with the paper.

The logit for the i-th class can be expressed by Eq. (7) and Eq. (8) in the paper (in which ''x_k \cdot m_i'' can be replaced by s_k^i ), so the form of the second term of Eq. (8) is equal to the line 26 at "pipeline/csra.py''.

Best,
Authors

**Comment by AlphaPlusTT** [comment]
Thanks for your reply. I try to use your method for pedestrian attribute recognition task with dataset rapv2, but the CSRA dose not work. Do you have any suggestions?

**Comment by abhigoku10** [comment]
@AlphaPlusTT i am also trying the same on PETA dataset and facing error , what error are you facing ? @Kevinz-code pls do share ur thoughts

**Comment by Kevinz-code** [comment]
@abhigoku10 @AlphaPlusTT 
Thanks for reading.

1. Actually we didn't run CSRA model on rapv2 and PETA, which weren't the popular multi-label recognition datasets in relevant literatures.  

2. We are not sure what "error" or "does not work" in your own implementation. Say, does the improvement of CSRA is low in comparison with the baseline method? It is advised that you provide the training details like model name, data aug, hyper-params, mAP, etc.

3. In our implementation in Wider-Attribute, the improvement is consistent by using VIT models, you can try VIT models and use hyper-parameters similar to our code as well.

Best,
